spec/moneta/proxies/weak_each_key/weak_each_key_spec.rb

```diff
@@ -9,8 +9,6 @@
   moneta_build do
     Moneta.build do
       use :WeakEachKey
-      # use :WeakIncrement
-      # use :WeakCreate
```

Comment:
I guess these comments were committed by accident? Can we remove them please?

Reply:
right, I did remove them :)